python/ray/ml/tests/test_horovod_trainer.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+import ray
+from ray.ml.train.integrations.horovod import HorovodTrainer
+from ray.train.examples.horovod.horovod_example import train_func as hvd_train_func
```

Comment:
Could you actually copy over the example to ray.ml.examples and have it use HorovodTrainer? Also I think it would be cool to also show the full predictor workflow as well (it would just be using TorchPredictor).